ClientLib/src/main/java/com/o3dr/services/android/lib/drone/connection/ConnectionParameter.java

```diff
@@ -277,7 +277,7 @@ public static ConnectionParameter newSoloConnection(String ssid, @Nullable Strin
             eventsDispatchingPeriod);
     }
 
-    private ConnectionParameter(@ConnectionType.Type int connectionType, Bundle paramsBundle){
+    public ConnectionParameter(@ConnectionType.Type int connectionType, Bundle paramsBundle){
```

Comment:
@mariansoban Thanks. I can revert this change for now and we can look at fixing the test cases